### ISSUE TITLE
Weight Decay

### ISSUE THREAD

**Issue by rohanchabra** [JiaRenChang/PSMNet#6]
Just a question if really weight decay is put to zero in optimization?

**Comment by JiaRenChang** [comment]
@rohanchabra 
Yes, we used the default weight decay (= 0) of Adam in the Pytorch.
